### PR TITLE
ES: UI

### PR DESCRIPTION
**Pull request by NozomiRem**

### CODE REVIEW

File: UI/ui_item.csv [hunk 1]
```diff
@@ -9401,163 +9401,163 @@ MenuGuide#9,"""<c f0df60>Al comerciar con FUN, puede comprar varios objetos. Por
...
+Name_002#0,"""Mayor probabilidad de objetos no identificados """
+Name_003#0,"""Restauracion automatica HP """
+Name_004#0,"""Bonificación en objeto curativo"""
+Name_005#0,"""Niega aleatoriamente el daño físico. """
+Name_006#0,"""Niega aleatoriamente el daño Tech."""
+Name_007#0,"""Previene estado quemar """
+Name_008#0,"""Previene la congelacion """
+Name_009#0,"""Previene Parálisis """
+Name_010#0,"""Previene Mirage """
+Name_011#0,"""Previene el Panico" ""
```
Comment (DabirSA) [inline UI/ui_item.csv]: You have a space between your ending quotes here.

Comment (NozomiRem) [reply UI/ui_item.csv]: Thx

File: UI/ui_item.csv [hunk 2]
```diff
@@ -9401,163 +9401,163 @@ MenuGuide#9,"""<c f0df60>Al comerciar con FUN, puede comprar varios objetos. Por
...
-StarTotalCashTitle#0,"""合計スター"""
-StarInsufficient#0,"""所持スターが不足しています。"""
-NoChangeItemMsg#0,"""購入できるアイテムがありません。"""
-NoEventMsg#0,"""賞品ショップの利用期限が過ぎているため、<br>購入できませんでした。"""
-EtcErrorMsg#0,"""購入できませんでした。<br>しばらく時間をおいてから実行して下さい。"""
+StarShopTitle#0,"""Tienda de premios"""
+StarMenuTitle#0,"""Lista de premios"""
+StarMenuGuide#0,"""<c f0df60>Puedes comprar artículos usando Star. Seleccione los artículos para comprar.<c>"""
+StarDenomination#0,"""Star"""
+StarCashTitle#0,"""Stars Actual"""
```
Comment (TheBlazter) [inline UI/ui_item.csv]: Stars Actuales

File: UI/ui_item.csv [hunk 3]
```diff
@@ -9401,163 +9401,163 @@ MenuGuide#9,"""<c f0df60>Al comerciar con FUN, puede comprar varios objetos. Por
...
+StarTotalCashTitle#0,"""Stars Total"""
+StarInsufficient#0,"""Insuficuentes Stars"""
```
Comment (TheBlazter) [inline UI/ui_item.csv]: Stars Insuficientes

File: UI/ui_title_text.csv [hunk 4]
```diff
@@ -441,28 +441,28 @@ Enquete_Answer10#0,"""10. Otros"""
...
-WindowSetting_Text#0,"""ゲーム中に表示される<br>ウィンドウと文字のサイズを<br>設定してください。"""
-WindowSetting_Button_S#0,"""スモール"""
-WindowSetting_Button_M#0,"""ミドル"""
-WindowSetting_Button_L#0,"""4K-スモール"""
-WindowSetting_Button_4KS#0,"""4K-ミドル"""
-WindowSetting_Button_4KM#0,"""4K-ラージ"""
+Comeback_Thanks_mes#0,"""Gracias por volver.<br>Acepta $(0) SG como agradecimiento.<br><br>¡Esperamos que sigas disfrutando de PSO2!"""
+Comeback_IconConfig_Title#0,"""Confirmación de Visualización del Icono"""
+Comeback_IconConfig_mes#0,"""¡Obtuvo una bonificación de jugador que regresa!<br>Durante 30 días, se le otorgara un efecto de bonificación cada que este en una Party con otras personas.<br>Los siguientes efectos se activan durante la bonificación:<br><br><yellow>- Tri-Boost +50%<c><br><yellow>- Recompenzas dobles de regalo de misiones destacadas.<c><br><br>¿Te gustaría alertar a los demás que eres un jugador que regresa mostrando el [Icono de Jugador Regresado <pict Comeback>] junto a tu nombre?<br><br>※Las bonificaciones se aplicarán independientemente de la configuración.<br>※Puede activar o desactivar el Icono a través del menu de Configuración Básica."""
+ButtonReset_Title#0,"""Confirmar Botones Asignados."""
```
Comment (TheBlazter) [inline UI/ui_title_text.csv]: Confirmar asignación de botones

**Pull request merged**